library/prelude/path.red

```diff
@@ -34,6 +34,20 @@ def funext
   λ i x →
   p _ i
 
+def happly 
+  (A : type) 
+  (B : A → type) 
+  (f g : (x : A) → B x) 
+  (p : path ((x : A) → B x) f g) 
+  : (x : A) → path (B x) (f x) (g x) = 
+  λ x i → p i x  
```

Comment:
In Cubical Thought, something like this should just be inlined; then we don't need to pass all these extra arguments to `happly` just to get a function application.

Reply:
I think I'm not actually using this anywhere, just added it for symmetry with `funext`.